robofleet_status/CMakeLists.txt

```diff
@@ -0,0 +1,211 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(robofleet_status)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  # ***add your package(s) containing custom message definitions like power info or location***
+  example_robot
+)
+
+## System dependencies are found with CMake's conventions
```

Comment:
we can probably clean up the commented out sections of this CMakeLists? It looks like this is copied from a template so maybe its fine to leave as-is though